templates/zerver/integrations/hubot.md

```diff
@@ -18,8 +18,8 @@ Use Hubot to execute scripts and commands within Zulip!
    information of the bot you created, by running:
 
     ```
-    export HUBOT_ZULIP_SITE="{{ api_url }}"
-    export HUBOT_ZULIP_BOT="hubot-bot@example.com"
+    export HUBOT_ZULIP_SITE="{{ zulip_url }}"
```

Comment:
Actually, shoot, have you tested this? This looks like a functional change, removing `/api/` from the URL -- are you sure this still works?